core/src/test/java/org/elasticsearch/search/suggest/AbstractSuggestionBuilderTestCase.java

```diff
@@ -135,13 +135,17 @@ public void testSerialization() throws IOException {
      */
     protected SB randomTestBuilder() {
         SB randomSuggestion = randomSuggestionBuilder();
-        maybeSet(randomSuggestion::text, randomAsciiOfLengthBetween(2, 20));
+        setCommonPropertiesOnRandomBuilder(randomSuggestion);
```

Comment:
True. It would be nice if we could do it in a single place, but since these are static methods that are called elsewhere, its the only option. I'll remove it from the `randomTestBuilder` method.